### PR TITLE
fix updateStockSelection function 

### PR DESCRIPTION
**Pull request by switch-cheehau**
refer to #240

### CODE REVIEW

**Comment by aimeos** [comment]
There will be an additional problem now because the product code is only unique per site. You should still use the product ID to fetch the product and compare the codes. If the don't match, get the variant article that's attached to that product.

**Comment by switch-cheehau** [comment]
> There will be an additional problem now because the product code is only unique per site. You should still use the product ID to fetch the product and compare the codes. If the don't match, get the variant article that's attached to that product.

Understand, i tried to update  the codes, can you help to review?
I've tested manually, the stocklevel will deduct normally for selection as well as its variant article.

I don't know how to adapt the tests, might need your advice on it.

File: controller/common/src/Controller/Common/Order/Standard.php [hunk 1]
```diff
@@ -430,45 +430,53 @@ protected function updateStockBundle( string $prodId, string $stockType )
 	/**
 	 * Updates the stock levels of selection products for a specific type
 	 *
-	 * @param string $prodId Unique product ID
-	 * @param string $stocktype Unique stock type
+	 * @param \Aimeos\MShop\Order\Item\Base\Product\Standard $item
 	 * @return \Aimeos\Controller\Common\Order\Iface Order controller for fluent interface
 	 */
-	protected function updateStockSelection( string $prodId, string $stocktype )
+	protected function updateStockSelection( $item )
```
Comment (aimeos) [inline controller/common/src/Controller/Common/Order/Standard.php:436]: Can you add the parameter type `\Aimeos\MShop\Order\Item\Base\Product\Iface` too?

**Comment by switch-cheehau** [comment]
still have a bug. The quantity does not -1 but become zero no matter how many stocks left.

**Comment by switch-cheehau** [comment]
Sorry, i misunderstand the concept. Will close the this PR.

However the issue still remain for selection product where it does not deduct the stocklevel

**Pull request closed without merging**